calibr8/utils.py

```diff
@@ -43,8 +44,10 @@ def __getattr__(self, attr):
 try:
     import pymc as pm
 
+    DistributionType: TypeAlias = Optional[pm.Distribution]
     HAS_PYMC = True
 except ModuleNotFoundError:
+    DistributionType: TypeAlias = Literal[None]  # type: ignore
```

Comment:
mypy doesn't like types to be redefined, but this is the only way I'm aware of to do type hinting with optional dependencies

Reply:
Okay, give me a little help here. The TypeAlias is actually defining attributes here?

Reply:
`TypeAlias` is just the type hint to let mypy know that `DistributionType` is an alias for a type - in this case `Literal[None]`

does that answer your question?